common/api.js

```diff
@@ -228,6 +231,9 @@ API.api = {
                     var userID = theEvent.user;
                     var roomID = theEvent.room;
                     var userName = theEvent.name;
+                    if (config.display.metadataUsernameField && theEvent.metadata && theEvent.metadata[config.display.metadataUsernameField]) {
+                      userName = theEvent.metadata[config.display.metadataUsernameField];
```

Comment:
If the event already includes the licode username (set when creating the token), why to check it in the metadata?